adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package noop2
+
+import (
+	"fmt"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/golang/protobuf/proto"
+
+	adapter "istio.io/mixer/pkg/adapter"
+	sample_report "istio.io/mixer/pkg/templates/sample/report"
+)
+
+type (
+	builder struct{}
+)
+
+func (builder) Name() string { return "noop2" }
+func (builder) Description() string {
+	return "An adapter that does nothing, just echos the calls made from mixer"
+}
+func (builder) Close() error { return nil }
+
+func (builder) ValidateConfig(msg proto.Message) error {
```

Comment:
@mandarjog Given the validation model for Galley, is this still how we want validation to be done in adapters?